test/source/tests/gmail.ts

```diff
@@ -392,6 +392,17 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       await gmailPage.close();
     }));
 
+    ava.default(`mail.google.com - render plain text for "message" attachment (which has plain text)`, testWithBrowser('ci.tests.gmail', async (t, browser) => {
+      const gmailPage = await openGmailPage(t, browser);
+      await gotoGmailPage(gmailPage, '/184a87a7b32dd009');
```

Comment:
FMfcgzGrbHrBdFGBXqpFZvSkcQpKkvrM
?

Reply:
sorry? why do we need that id?
`184a87a7b32dd009` is gmail thread which contains plain text + plain `message` attachment?

Reply:
sorry? why do we need that id?
`184a87a7b32dd009` is gmail thread which contains plain text + plain `message` attachment?

Reply:
oh interesting. Then why do we use this type of link for `mail.google.com - encrypted text inside "message" attachment` test?

Reply:
Also, please add `eml` from the live account to the repo

Reply:
I'm not sure what you mean.
We test `FMfcgzGrbHprlHvtTJscCJQpZcqrKQbg` thread for `mail.google.com - encrypted text inside "message" attachment`  test.
And we test `184a87a7b32dd009` thread for `mail.google.com - render plain text for "message" attachment (which has plain text)` test

Reply:
Ohhh.. you are right.
we need to use `FMfcgzGrbHrBdFGBXqpFZvSkcQpKkvrM` for `mail.google.com - render plain text for "message" attachment (which has plain text) `.
I was confused with mock thread id.

Reply:
If I enter https://mail.google.com/mail/u/0/#inbox/184a87a7b32dd009, it redirects to https://mail.google.com/mail/u/0/#inbox/FMfcgzGrbHrBdFGBXqpFZvSkcQpKkvrM though

Reply:
Therefore both ids are fine

Reply:
This is an interesting fact!

Reply:
Let's merge it as it is now